docs/reference/plugins/ant_plugin.rst

```diff
@@ -0,0 +1,98 @@
+.. _rockcraft_ant_plugin:
```

Comment:
My problem with this approach is that docs won't be able to build successfully because of the 2 craft-parts pages (ant_plugin.rst and maven_plugin.rst) that are no longer referenced here in rockcraft. One way to fix that is to follow the approach done [here](https://github.com/canonical/rockcraft/blob/main/docs/reference/plugins/poetry_plugin.rst) where you pick what part of the craft-parts doc you want to include/exclude.

Reply:
here's the sphinx error:
```
pickling environment... done
checking consistency... /home/docs/checkouts/readthedocs.org/user_builds/canonical-rockcraft/checkouts/752/docs/common/craft-parts/reference/plugins/ant_plugin.rst: WARNING: document isn't included in any toctree
/home/docs/checkouts/readthedocs.org/user_builds/canonical-rockcraft/checkouts/752/docs/common/craft-parts/reference/plugins/maven_plugin.rst: WARNING: document isn't included in any toctree
```